DOC/website: Move citiation information from README to separate file and subsection "citing" (#4784)
GenericMappingTools/pygmt: doc/citing.md

# Citing PyGMT

If you used PyGMT in your research please consider citing PyGMT. You can either cite the

> [PyGMT paper](https://doi.org/10.1029/2026GC013105) published in *Geochemistry, Geophysics, Geosystems*

Reviewer: Maybe shorten it a little bit?
```suggestion
[PyGMT paper](https://doi.org/10.1029/2026GC013105)
```

Author: Removed in commit https://github.com/GenericMappingTools/pygmt/pull/4784/commits/eadce0b1214f0569a63b8d0481ded1349bf20a3a.

> of the PyGMT software. To cite a specific version of PyGMT, go to the Zenodo archive, select the specific

Reviewer: I still think we should move this sentence to the end. It does not make sense to discuss a specific version before referencing the latest release.

Author: Done in commit https://github.com/GenericMappingTools/pygmt/pull/4784/commits/45480c966e48846a92a865091c186cb55eab1612 (hopefully now correct).

> version of PyGMT, change to it on Zenodo and use the "Export to BibTeX" function.

Reviewer: It's unclear what "change to it on Zenodo" means. 

Maybe revert to the original text 

> To cite a specific
version of PyGMT, go to the Zenodo page at <https://doi.org/10.5281/zenodo.3781524>, select the specific versio, 
and use the "Export to BibTeX" function there.

Author: See commit https://github.com/GenericMappingTools/pygmt/pull/4784/commits/c63f03613346dfdcb48653292c88b5e88d1e1cb2.

It is also strongly recommended to cite the [GMT 6 paper](https://doi.org/10.1029/2019GC008515) (which
PyGMT wraps around). Note that some modules like `dimfilter`, `surface`, and `x2sys` also have their dedicated
citations. Further information for all these can be found at <https://www.generic-mapping-tools.org/cite>.

The BibTeX entry for the PyGMT paper is:
```
@article{pygmt_2026_2026GC013105,
  author       = {Tian, Dongdong and
                  Fröhlich, Yvonne and
                  Leong, Wei Ji and
                  Grund, Michael and
                  Schlitzer, William and
                  Jones, Max and
                  Uieda, Leonardo and
                  Luis, Joaquim M. F.},
  title        = {{PyGMT: Bridging Python and the Generic Mapping Tools for Geospatial Visualization and Analysis}},
  journal      = {Geochemistry, Geophysics, Geosystems},
  year         = 2026,
  volume       = {27},
  number       = {7},
  pages        = {e2026GC013105},
  doi          = {10.1029/2026GC013105},
  url          = {https://doi.org/10.1029/2026GC013105},
}
```

> The BibTeX entry for the latest Zenodo archive of the PyGMT software is:

Reviewer: The term "the latest Zenodo" may be inaccurate. Imagine that when we have v0.25.0, but someone who visits the v0.20.0 documentation may think this is the correct BibTeX to use.

Perhaps we should add the PyGMT version information to clarify that it's the bibtex for a specific version, rather than "the latest version".

Author: Just added the version number manually. As it's hard-coded at the moment, we need to not forget to update it when making the release:

> The BibTeX entry for the latest Zenodo archive, i.e., release v0.19.0, of the PyGMT software is:

Or we add the BibTeX for the conceptual DOI? Then it would be consistent to the Zenodo archive link above.

Reviewer: > Or we add the BibTeX for the conceptual DOI? Then it would be consistent to the Zenodo archive link above.

See https://zenodo.org/help/versioning.

> You should normally always use the DOI for the specific version of your record in citations. This is to ensure that other researchers can access the exact research artefact you used for reproducibility. By default, we use the specific version to generate citations.
> 
> You can use the Concept DOI representing all versions in citations when it is desirable to cite an evolving research artifact, without being specific about the version.

Author: OK. Then keep the latest version for the BibTex entery.

```
@software{pygmt_2026_19398871,
  author       = {Tian, Dongdong and
                  Leong, Wei Ji and
                  Fröhlich, Yvonne and
                  Grund, Michael and
                  Schlitzer, William and
                  Jones, Max and
                  Toney, Liam and
                  Yao, Jiayuan and
                  Tong, Jing-Hui and
                  Magen, Yohai and
                  Materna, Kathryn and
                  Belem, Andre and
                  Newton, Tyler and
                  Anant, Abhishek and
                  Ziebarth, Malte and
                  Quinn, Jamie and
                  He, Xingchen and
                  Uieda, Leonardo and
                  Wessel, Paul},
  title        = {{PyGMT: A Python interface for the Generic Mapping Tools}},
  month        = jul,
  year         = 2026,
  publisher    = {Zenodo},

>   version      = {0.19.0},

Author: Should we include the "v"? At Zenodo it is included, please see https://doi.org/10.5281/zenodo.3781524:

> Tian, D., Leong, W. J., Fröhlich, Y., Grund, M., Schlitzer, W., Jones, M., Toney, L., Yao, J., Tong, J.-H., Magen, Y., Materna, K., Belem, A., Newton, T., Anant, A., Ziebarth, M., Quinn, J., He, X., Uieda, L.& Wessel, P. (2026). PyGMT: A Python interface for the Generic Mapping Tools (Version v0.19.0) [Computer software]. Zenodo. https://doi.org/10.5281/zenodo.19398871

Reviewer: In our Zenodo records, we set the version field to "v0.19.0" rather than "0.19.0", but Zenodo suggests using a semantic version string.

<img width="1700" height="240" alt="image" src="https://github.com/user-attachments/assets/026c9c71-fd8a-4946-8743-2b7932d3052e" />

According to the semver site, v0.19.0 is NOT a semantic version, 0.19.0 is (xref: https://semver.org/#is-v123-a-semantic-version). In other words, we should use 0.19.0  in the version field.

Author: OK. This means we should update the metadata of the Zenodo uploads.

Reviewer: I think so. @weiji14 any comments on it?

Author: I believe we already had some similar discussions on this "v" during one release but could not find it at the moment in the release issues or changelog PRs.

_Edit_: I found it https://github.com/GenericMappingTools/pygmt/pull/2272#issuecomment-1364703044 and https://github.com/GenericMappingTools/pygmt/pull/2272#issuecomment-1364836487. For release 0.8.0, we decided to drop the "v" in the BibTeX entry in the README to be consistent with the CITIATION.cff file (https://github.com/GenericMappingTools/pygmt/pull/2272/changes#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168f).

Currently we use the "v" in the GitHub releases and tags (https://github.com/GenericMappingTools/pygmt/tags) and the file name for the [Zenodo archive](https://zenodo.org/records/19398871) as well as in the table for the [Minimum Supported Versions](https://www.pygmt.org/dev/minversions.html). Probably need to add some note to the [Maintainers Guide](https://www.pygmt.org/dev/maintenance.html#making-a-release).

Reviewer: Maybe open an issue so that we can discuss it in detail

Author: Agree, better to have a separate issue for more detailed discussions, please see #4790.

of the PyGMT software.

It is also strongly recommended to cite the [GMT 6 paper](https://doi.org/10.1029/2019GC008515) (which PyGMT
wraps around). Note that some modules like `dimfilter`, `surface`, and `x2sys` also have their dedicated
citations. Further information for all these can be found at <https://www.generic-mapping-tools.org/cite>.

The BibTeX entry for the PyGMT paper is:
```
@article{pygmt_2026_2026GC013105,
  author       = {Tian, Dongdong and
                  Fröhlich, Yvonne and
                  Leong, Wei Ji and
                  Grund, Michael and
                  Schlitzer, William and
                  Jones, Max and
                  Uieda, Leonardo and
                  Luis, Joaquim M. F.},
  title        = {{PyGMT: Bridging Python and the Generic Mapping Tools for Geospatial Visualization and Analysis}},
  journal      = {Geochemistry, Geophysics, Geosystems},
  year         = 2026,
  volume       = {27},
  number       = {7},
  pages        = {e2026GC013105},
  doi          = {10.1029/2026GC013105},
  url          = {https://doi.org/10.1029/2026GC013105},
}
```

> The BibTeX entry for the latest Zenodo archive, i.e., release v0.19.0, of the PyGMT software is:

Author: Should we keep the "v" (corresponding the [release on GitHub](https://github.com/GenericMappingTools/pygmt/releases#release-v0.19.0)) or remove it (to be consistent with the Zenodo upload after updating the metadata)?

```suggestion
The BibTeX entry for the latest Zenodo archive, i.e., release 0.19.0, of the PyGMT software is:
```

```
@software{pygmt_2026_19398871,
  author       = {Tian, Dongdong and
                  Leong, Wei Ji and
                  Fröhlich, Yvonne and
                  Grund, Michael and
                  Schlitzer, William and
                  Jones, Max and
                  Toney, Liam and
                  Yao, Jiayuan and
                  Tong, Jing-Hui and
                  Magen, Yohai and
                  Materna, Kathryn and
                  Belem, Andre and
                  Newton, Tyler and
                  Anant, Abhishek and
                  Ziebarth, Malte and
                  Quinn, Jamie and
                  He, Xingchen and
                  Uieda, Leonardo and
                  Wessel, Paul},
  title        = {{PyGMT: A Python interface for the Generic Mapping Tools}},
  month        = jul,
  year         = 2026,
  publisher    = {Zenodo},
  version      = {0.19.0},
  doi          = {10.5281/zenodo.19398871},
  url          = {https://doi.org/10.5281/zenodo.19398871}
}
```

> To cite a specific version of PyGMT, go to the Zenodo archive, select the specific version, and use the

Reviewer: better to add a link for "Zenodo aechive"

Author: Zenodo link added in commit https://github.com/GenericMappingTools/pygmt/pull/4784/commits/bb65027d4c7def56e9611c79e20e1031703bac85.

# Citing PyGMT

If you used PyGMT in your research please consider citing PyGMT. You can either cite the
[PyGMT paper](https://doi.org/10.1029/2026GC013105) published in *Geochemistry, Geophysics, Geosystems*
or the Zenodo archive of the [PyGMT software](https://doi.org/10.5281/zenodo.3781524).

To cite a specific version of PyGMT, go to our Zenodo page at <https://doi.org/10.5281/zenodo.3781524>
and use the "Export to BibTeX" function there. It is also strongly recommended to cite
the [GMT 6 paper](https://doi.org/10.1029/2019GC008515) (which PyGMT wraps around). Note

> that some modules like `dimfilter`, `surface`, and `x2sys` also have their dedicated

Reviewer: Perhaps we should add a reference section on the API documentation page of these methods?

Author: I added the publications mentioned on the GMT website https://www.generic-mapping-tools.org/cite/ to the related API doc pages, see commit https://github.com/GenericMappingTools/pygmt/pull/4784/commits/a26fa4a120558e6aed4c23dbd87d313159123e8f.

> or the Zenodo archive of the [PyGMT software](https://doi.org/10.5281/zenodo.3781524).
> 
> To cite a specific version of PyGMT, go to our Zenodo page at <https://doi.org/10.5281/zenodo.3781524>
> and use the "Export to BibTeX" function there. It is also strongly recommended to cite
> the [GMT 6 paper](https://doi.org/10.1029/2019GC008515) (which PyGMT wraps around). Note
> that some modules like `dimfilter`, `surface`, and `x2sys` also have their dedicated
> citations. Further information for all these can be found at <https://www.generic-mapping-tools.org/cite>.

Reviewer: ```suggestion
If you used PyGMT in your research please consider citing PyGMT. You can either cite the
[PyGMT paper](https://doi.org/10.1029/2026GC013105) published in *Geochemistry, Geophysics, Geosystems*
or the Zenodo archive of the [PyGMT software](https://doi.org/10.5281/zenodo.3781524).
It is also strongly recommended to cite
the [GMT 6 paper](https://doi.org/10.1029/2019GC008515) (which PyGMT wraps around). Note
that some modules like `dimfilter`, `surface`, and `x2sys` also have their dedicated
citations. Further information for all these can be found at <https://www.generic-mapping-tools.org/cite>.
```

Move the following sentence to the end?

> To cite a specific version of PyGMT, go to our Zenodo page at <https://doi.org/10.5281/zenodo.3781524>
and use the "Export to BibTeX" function there.

Author: I made a change in commit https://github.com/GenericMappingTools/pygmt/pull/4784/commits/a1026583b07583f52cc89860b692f9c725e910a4. Is this what you propose?

The BibTeX entry for the PyGMT paper is:
```
@article{pygmt_2026_2026GC013105,
  author       = {Tian, Dongdong and
                  Fröhlich, Yvonne and
                  Leong, Wei Ji and
                  Grund, Michael and
                  Schlitzer, William and
                  Jones, Max and
                  Uieda, Leonardo and
                  Luis, Joaquim M. F.},
  title        = {{PyGMT: Bridging Python and the Generic Mapping Tools for Geospatial Visualization and Analysis}},
  journal      = {Geochemistry, Geophysics, Geosystems},
  year         = 2026,
  volume       = {27},
  number       = {7},
  pages        = {e2026GC013105},
  doi          = {10.1029/2026GC013105},
  url          = {https://doi.org/10.1029/2026GC013105},
}
```
The BibTeX entry for the latest Zenodo archive of the PyGMT software is:
```
@software{

>   pygmt_2026_19398871,

Reviewer: More consistent with the other BibTeX entry:
```suggestion
@software{pygmt_2026_19398871,
```

# Citing PyGMT

> PyGMT is a community developed project. See the

Reviewer: We probably can remove this paragraph, but let's revisit it later

Author: Agree, it does not really fit to the citing section. Maybe we can move and mention this to the [PyGMT team](https://www.pygmt.org/dev/team.html) page?

Author: Actually we already mention the `AUTHORS.md` file on the "PyGMT Team" page:

https://github.com/GenericMappingTools/pygmt/blob/7620be6091b50ed6f767e696aebbe180bd33171c/doc/team.md?plain=1#L5-L10

Thus, I removed the paragraph:

https://github.com/GenericMappingTools/pygmt/blob/7620be6091b50ed6f767e696aebbe180bd33171c/README.md?plain=1#L117-L121

from the "Citing PyGMT page" and included it into the paragraph on the "PyGMT Team" page (see commit https://github.com/GenericMappingTools/pygmt/pull/4784/commits/e97861b2376fcddccb78ed199f160c7046034eee).
